Panic in datatransceiver.go

Revision: `v4.1.0-beta5`

```
panic: send on closed channel

goroutine 139467 [running]:
github.com/peer-calls/peer-calls/server.(*DataTransceiver).start(0xc0027580c0)
	/home/runner/work/peer-calls/peer-calls/server/datatransceiver.go:128 +0x37d
created by github.com/peer-calls/peer-calls/server.NewDataTransceiver
	/home/runner/work/peer-calls/peer-calls/server/datatransceiver.go:63 +0x2d6
```

--- a/server/datatransceiver.go
+++ b/server/datatransceiver.go
@@ -127,6 +127,8 @@ func (d *DataTransceiver) start() {
 
 				msgFuture.errCh <- errors.Trace(err)
 			}
+
+			close(msgFuture.errCh)
 		case <-d.teardownChan:
 			return
 		}
@@ -158,9 +160,8 @@ func (d *DataTransceiver) Send(message webrtc.DataChannelMessage) <-chan error {
 	}:
 	case <-d.torndownChan:
 		errCh <- errors.Trace(io.ErrClosedPipe)
+		close(errCh)
 	}
-
-	close(errCh)
 
 	return errCh
 }

